CompareComplyV1.playground/Pages/Batches.xcplaygroundpage/Contents.swift

```diff
@@ -0,0 +1,69 @@
+
+import CompareComplyV1
```

Comment:
I am very surprised not to see:
```
import PlaygroundSupport

// Enable support for asynchronous completion handlers
PlaygroundPage.current.needsIndefiniteExecution = true
```
on every playground page.  I thought this was required for our completion handlers to work.

If this isn't needed, we should probably go back and remove it from all the other playgrounds.  But please investigate, because I really think it is necessary.

Reply:
Forgot to comment on this. Yes, I confirmed that all of the C&C playgrounds execute successfully without `PlaygroundPage.current.needsIndefiniteExecution = true`. We can probably remove them from the other playgrounds too.